src/Common/src/Microsoft/Win32/SafeHandles/SafeBCryptAlgorithmHandle.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using NTSTATUS = Interop.BCrypt.NTSTATUS;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeBCryptAlgorithmHandle : SafeBCryptHandle
```

Comment:
SafeBCryptAlgorithmHandle.Windows.cs; if I understand the naming convention in this directory :).